### PR TITLE
feature/COR-1548-tests-page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard26**
## Summary

* added new `Tests` route component;
* updated `PositivelyTestedPeople` route component to remove self tests chart;
* updated reverse router to add new Tests route ('/landelijk/testen') and clean up;
* updated `NlLayout` component to add new Tests route ('/landelijk/testen') to sidebar;
* moved Sanity keys using DELETE and ADD;
* updated types;

### Notes
* This pull request does include the removal of the 'Self-reported positive corona test results' from the `PositivelyTestedPeople` route in order to also clean up associated Sanity keys. However, this pull request does not include any additional archiving of this route as that is to be picked up as part of COR-1547.
* A new page within Sanity has been created for the keys associated to the new `Tests` route. This can be found in Sanity > Dashboard Pagina's > Testen > nl > various keys.

#### Before
No before screenshots apply as this is a completely new page.

#### After
<details>
<summary>Toggle after screenshots</summary>

COR-1548_after
</details>

### CODE REVIEW

**Review by VWSCoronaDashboard28 — DISMISSED** [review]

File: packages/app/src/domain/layout/logic/types.ts [hunk 1]
```diff
@@ -27,14 +27,15 @@ export type NlItemKeys =
   | 'elderly_at_home'
   | 'general_practitioner_suspicions'
   | 'hospitals_and_care'
-  | 'patients'
   | 'infectious_people'
   | 'mortality'
   | 'nursing_home_care'
+  | 'patients'
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/domain/layout/logic/types.ts:33]: Nice sorting of items!

File: packages/app/src/domain/layout/logic/use-sidebar.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const mapKeysToReverseRouter = {
   // Still the nursing home care name is used because of legacy naming inside of sanity's lokalize texts.
   nursing_home_care: 'kwetsbareGroepen',
   positive_tests: 'positiefGetesteMensen',
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/domain/layout/logic/use-sidebar.tsx:30]: Are you going to sort the list (including patients) here too?
Of course feel free to resolve, absolutely nonblocking!

Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/layout/logic/use-sidebar.tsx:30]: Nice Adjustment! I will resolve this!

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice! Looking good!

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Pull request merged**